### DISCUSSION TITLE
Where is Servo.cpp code for ESP8266

### DISCUSSION THREAD

**Discussion by HAJM** [esp8266/Arduino#7916] [Q&A] [answered]
I can find the Servo.h file for servo support on an ESP8266 processor at
     https://github.com/gbrault/esp8266-Arduino/blob/master/libraries/Servo/src/Servo.h
but I cannot find the corresponding Servo.cpp file.  Does anyone know where it is or, better still, does anyone know how I can download the zip containing both of the files?

**Comment by d-a-v** [comment, marked as answer]
Wrong library, please use the official one

https://github.com/esp8266/Arduino/tree/master/libraries/Servo/src

https://github.com/esp8266/Arduino/ (not https://github.com/gbrault/esp8266-Arduino)

**Reply by HAJM** [reply]
Thank you for the links, which have solved my problem, I have marked your reply as an answer, to end the discussion.
